Dockerfile

```diff
@@ -1,21 +1,20 @@
 FROM registry.hub.docker.com/library/debian:stable-slim
 
-LABEL Maintainer="software-embedded-platform@ultimaker.com" \
+LABEL Maintainer="software-embedded@ultimaker.com" \
       Comment="Ultimaker kernel build environment"
 
-RUN dpkg --add-architecture armhf && \
+RUN dpkg --add-architecture arm64 && \
     apt-get update && \
     apt-get install -q -y --no-install-recommends \
         build-essential \
+        gcc-aarch64-linux-gnu \
         cmake \
-        crossbuild-essential-armhf \
+        crossbuild-essential-arm64 \
         git \
-        libevdev-dev:armhf \
+        libevdev-dev:arm64 \
+        make \
```

Comment:
I can't really find it here now, but instead of running `make` directly, you could also decide to run `cmake --build`. Which does the same, but then you use the cmake platform to perform your target tasks.